docs/gatsbyjs.md

```diff
@@ -73,3 +73,19 @@ npm run develop
 
 This will run the Gatsby app with hot-reloading on `http://localhost:8000` and
 use `https://dev.telescope.cdot.systems` as your backend api.
+
+Please note `npm run develop` will detect 99% of any issues that may occur when working on the front-end. It is highly advised to run `npm run build` to verify there are no issues.
```

Comment:
Should you not specify this above instead of `npm run build`, since most people will want hot-reloading and automatic rebuilds vs a single static build?  I think `npm run build` is really only useful for deployment.

Reply:
I don't want people to use `npm run build` actually until at the very end of the development when they think everything is good to go. Hence this note. I remember working on a PR where nothing failed when I was developing with `npm run develop` but as soon as I tried to build, it failed.

Reply:
If we can move all discussion of `build` and `serve` to the very end, and make it clear what this is for, that's cool.  But I want to avoid confusing people with multiple options. Every choice we give someone is another person we'll have to help debug through their misunderstanding.

Let's have a single recommended workflow.